modules/ui/src/main/scala/lucuma/ui/primereact/RadioButtonView.scala

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import cats.Eq
+import cats.syntax.all._
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common.*
+import react.primereact.RadioButton
+
+import scalajs.js
+
+final case class RadioButtonView[V[_], A](
+  id:       NonEmptyString,
+  value:    A,                                         // The value this radio button will assign to the ExternalValue
+  view:     V[A],
+  label:    NonEmptyString,
+  name:     js.UndefOr[NonEmptyString] = js.undefined, // name of the "radio button group"
+  inputId:  js.UndefOr[String] = js.undefined,
+  disabled: js.UndefOr[Boolean] = js.undefined,
+  clazz:    js.UndefOr[Css] = js.undefined,
+  required: js.UndefOr[Boolean] = js.undefined,
+  onChange: js.UndefOr[(A, Boolean) => Callback] =
+    js.undefined, // shouldn't usually need to use this, the ViewLike is set internally.
```

Comment:
If you do, do you have to take care of setting the `ViewLike`?

Reply:
No, the onChange passed in the properties is called after the ViewLike is set.

Reply:
I guess it probably isn't necessary since you could use `.withOnMod`, but it's there.